packages/suite/src/components/wallet/CoinmarketLayout/components/AccountTransactions/components/SavingsTransaction/index.tsx

```diff
@@ -126,7 +130,10 @@ const SavingsTransaction = ({ trade, providers, account }: Props) => {
                         <Icon color={theme.TYPE_LIGHT_GREY} size={13} icon="ARROW_RIGHT" />
                     </Arrow>
                     <HiddenPlaceholder>
-                        {formatCryptoAmount(Number(receiveStringAmount))} {receiveCurrency}
+                        <FormattedCryptoAmount
```

Comment:
When the switch to `<FormattedCryptoAmount>` is done, we should get rid of the `formatCryptoAmount` function.

Reply:
Definitely, I will remove it when it's not needed.